.eslintrc.js

```diff
@@ -24,15 +24,16 @@ module.exports = {
       }
     ],
 
+    "@typescript-eslint/no-empty-function": "off",
     "@typescript-eslint/no-unused-vars": [
       "error",
       {
-        "argsIgnorePattern": "^_"
+        "args": "none"
```

Comment:
This one had to be changed since openapi appeared to be relying on the actual argument name (so `_log`  would break things, while `log` wouldn't).